server/views/pages/bookings/visit.njk

```diff
@@ -0,0 +1,92 @@
+{% set activeLink = 'bookings-link' %}
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = "Bookings" %}
```

Comment:
```suggestion
{% set pageTitle = "Visit booking details" %}
```

... perhaps then use this variable in line 13 for the `h1` (in fact we should probably do this elsewhere as a searate tweak as it looks like we're duplicating the `title` and `h1` text in most of the other templates!)